addons/web/static/tests/setup.js

```diff
@@ -246,13 +246,23 @@ function removeUnwantedAttrsFromTemplates(attrs) {
     function replaceAttr(attrName, prefix, element) {
         const attrKey = `${prefix}${attrName}`;
         const attrValue = element.getAttribute(attrKey);
-        element.removeAttribute(attrKey);
+        if (element.nodeName === "img") {
+            if (attrKey === "src") {
+                element.setAttribute(attrKey, "/web/image/web.image_placeholder");
+            } else if (attrKey === "t-att-src" && attrKey === "t-attf-src") {
+                element.setAttribute(attrKey, "'/web/image/web.image_placeholder'");
+            }
+        } else {
+            element.removeAttribute(attrKey);
+        }
         element.setAttribute(`${prefix}data-${attrName}`, attrValue);
     }
     const attrPrefixes = ["", "t-att-", "t-attf-"];
     for (const attrName of attrs) {
         for (const prefix of attrPrefixes) {
-            for (const element of templates.querySelectorAll(`*[${prefix}${attrName}]`)) {
+            for (const element of templates.querySelectorAll(
+                `*[${prefix}${attrName}]:not([${prefix}${attrName}='/web/image/web.image_placeholder'])`
+            )) {
```

Comment:
Why is that? It's ok to just replace by the same no?
Also good to have the data-src to be consistent in test assertions.

Reply:
without the `not` condition, it seams to infinity loop on some attributes.

Reply:
Would it be possible to put the condition on existence of `data-src` instead?